### ISSUE TITLE
After pressing TAB Keyboard focus is landing on "Web Browser" section but focus is not visible on it: A11y_WinForms.NETRuntime_CommonControl2_FocusVisible.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13655]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags
#A11ySev2; #A11yTCS; #WCAG2.4.7; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Focus Visible; #A11yMAS

### Environment Details
Application Name: WinForms Runtime
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061

### Repro Steps

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Common Control 2" button and hit ENTER key.
4. TAB till "Web Browser" section.
5. Observe that After pressing TAB Keyboard focus is landing on "Web Browser" section but focus is not visible on it.

### Actual Result
After pressing TAB Keyboard focus is landing on "Web Browser" section but focus is not visible on it.

### Expected Result
After pressing TAB Keyboard focus should be visible on Web Browser's interactive element which is Scroll bar of that section.

### User Impact
Keyboard user will not be able to see where keyboard focus is shifting after pressing TAB and cannot be able to access the data.

### Attachment

https://github.com/user-attachments/assets/5f1c50e9-e68e-4d3b-ad6e-2b86bba460d9

**Comment by Olina-Zhang** [comment]
Dup of the GH issue: https://github.com/dotnet/winforms/issues/11437, we updated the WebBrowser with a simple html doc to avoid web page in WebBrowser steals focus, see discussion in https://github.com/dotnet/winforms/issues/7337#issuecomment-1165930875

**Comment by merriemcgaw** [comment]
@Olina-Zhang  it doesn't look like the html doc is loading. Can you make sure that change got into the sample app in the repo?

**Comment by Olina-Zhang** [comment]
@merriemcgaw I forgot to include that html file for publish. Now publish it again, can get html loaded in webBrowser control

![Image](https://github.com/user-attachments/assets/52cebf38-7dfb-471e-80f4-1233a8c1d8cf)

@shindepra Can you please uninstall that test app and install the updated one from shared path again on your machine for testing?
